homeassistant/components/openweathermap/weather_update_coordinator.py

```diff
@@ -22,21 +31,29 @@
     ATTR_API_WIND_SPEED,
     CONDITION_CLASSES,
     DOMAIN,
+    FORECAST_MODE_DAILY,
+    FORECAST_MODE_HOURLY,
+    FORECAST_MODE_ONECALL_DAILY,
+    FORECAST_MODE_ONECALL_HOURLY,
 )
 
 _LOGGER = logging.getLogger(__name__)
 
-WEATHER_UPDATE_INTERVAL = timedelta(minutes=10)
+WEATHER_UPDATE_INTERVAL = timedelta(minutes=5)
```

Comment:
Decreasing the update interval doesn't make much sense for slowly changing data.

I'm not aware that OWM change their update frequency recently.

Reply:
I reduced it because I thought 10 mins was set to avoid hitting free API limits, but yeah makes sense.